src/App.js

```diff
@@ -1,4 +1,12 @@
 import React from "react";
+// import logo from "./logo.svg";
+// import { Counter } from "./store/counter/Counter";
+import "./App.css";
+
+
+function App() {
```

Comment:
-Remove the line 7 to 9. These lines are unnecessary and crash the app. It also prevents the PR from being deployed to vercel.
```suggestion

```